File: code-coaches/easy/paint-costs/PaintCosts.swift
func getTotalPaintCosts(numColors: Int) -> Int {
    let basePrice = 40.0
    let paintPrice = 5.0
    let taxPercent = 10.0

    let subtotal = basePrice + (paintPrice * colorsNumber)
    let total = ((subtotal / taxPercent) + subtotal).rounded(.up)

    return Int(total)
}

func main() {
    let numColors = Double(readLine()!)!
	print(getTotalPaintCosts(numColors: numColors))
}

main()

File: code-coaches/easy/paint-costs/PaintCosts.cs
using System;

public class PaintCosts
{
    static int GetTotalPaintCosts(int numColors)
    {
        double basePrice = 40.0;
        double paintPrice = 5.0;
        double taxPercent = 10.0;

        double subtotal = basePrice + (paintPrice * colorsNumber);
        int total = (int) Math.Ceiling((subtotal / taxPercent) + subtotal);

        return total;
    }

    static void Main(string[] args)
    {
        int numColors = Convert.ToInt32(Console.ReadLine());
        Console.WriteLine(GetTotalPaintCosts(numColors));
    }
}
